compiler/src/iree/compiler/Preprocessing/Common/PadLinalgOps.cpp

```diff
@@ -153,8 +155,12 @@ class PadMatmulOp : public OpInterfaceRewritePattern<linalg::LinalgOp> {
   int paddingSize;
 };
 
-class PadLinalgOpsPass : public PadLinalgOpsBase<PadLinalgOpsPass> {
+class PadLinalgOpsPass
+    : public iree_compiler::Preprocessing::impl::PadLinalgOpsPassBase<
+          PadLinalgOpsPass> {
 public:
+  using iree_compiler::Preprocessing::impl::PadLinalgOpsPassBase<
+      PadLinalgOpsPass>::PadLinalgOpsPassBase;
```

Comment:
BTW, while we are at this, this is a bit redundant. I think it would be more readable with:
```c++
struct PadLinalgOpsPass final
    : impl::PadLinalgOpsPassBase<PadLinalgOpsPass> {
  using impl::PadLinalgOpsPassBase<
      PadLinalgOpsPass>::PadLinalgOpsPassBase;
```
also elsewhere